config/set/naming/naming.yaml

```diff
@@ -1,3 +1,3 @@
 services:
```

Comment:
I noticed that there is one rule missing in this configuration file.
Rector\Naming\Rector\ClassMethod\RenameVariableToMatchNewTypeRector.

Do you want me to add it withing PR or let's create a new issue for it?

Reply:
I guess I know why it is missing there :) because this rules are applied in CI for Rector itself...
@TomasVotruba, let me know what is the correct solution here, please. To remove it from set, or to setup CI somehow to skip this rule?

Reply:
> Rector\Naming\Rector\ClassMethod\RenameVariableToMatchNewTypeRector.

I guess it still needs some work. Should be added and ideally, those cases fixed :+1:

Reply:
I wasn't that clear... At the moment, the CI fails because of the new rule that has been added to the set. I'm not sure this issue should be fixed.

Reply:
I get it. The code should always pass for Rector ruleset. If it's ignored, it kills the point of Rector rules helping us with code-reviews.

This should be enough to fix code locally:

```bash
bin/rector p --config rector-ci.yaml
```

Reply:
Thanks. Found an issue with the rule :) I'll change the PR status to Ready For Review once this rule is ready for your review.